packages/machine-extractor/src/toMachineConfig.ts

```diff
@@ -48,6 +49,13 @@ const parseStateNode = (
     config.type = astResult.type.value as any;
   }
 
+  if (isRootNode && t.isObjectExpression(astResult.context?.node)) {
```

Comment:
what would happen with `context: () => ({})`? I mean, not here - but further down the road

Reply:
We don't support lazy context in Studio yet so that's ignored for now.

Reply:
Are there any downsides to that? What if we export the code after some changes? Would the lazy context be completely lost in such a scenario?

Reply:
Extracting lazy context doesn't benefit us other than not being lossy to that data. It won't be editable, won't be rendered in context panel and won't certainly be integrated with context schema. I'm not against supporting it if it's just a read-only context expression.

Reply:
>  I'm not against supporting it if it's just a read-only context expression.

Yeah, I was thinking about something like that for now.

Reply:
Done.